src/libs/WorkspacesSettingsUtils.ts

```diff
@@ -54,11 +55,12 @@ Onyx.connect({
 });
 
 /**
- * @param report
- * @returns BrickRoad for the policy passed as a param
+ * @param altReportActions Replaces (local) allReportActions used within (local) function getWorkspacesBrickRoads
+ * @returns BrickRoad for the policy passed as a param and optionally actionsByReport (if passed)
  */
-const getBrickRoadForPolicy = (report: Report): BrickRoad => {
-    const reportActions = allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`] ?? {};
+const getBrickRoadForPolicy = (report: Report, altReportActions?: OnyxCollection<ReportActions>): BrickRoad => {
+    const finalReportActions = isEmptyObject(altReportActions) ? allReportActions : altReportActions;
```

Comment:
@iwiznia Correct, are you suggesting doing this instead ?

`const reportActions = (altReportActions ?? allReportActions)?.['${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}'] ?? {};`

Reply:
`altReportActions ?? allReportActions` I assume this is a typo?

Reply:
`altReportActions` is the alternative reportActions coming from param and `allReportActions` are the local reportActions coming from Onyx.connect.

`altReportActions ?? allReportActions` suggestion was made because you pointed out that `isEmptyObject(altReportActions) ? allReportActions : altReportActions` is not necessary since if `altReportActions` is not passed it means it's undefined / null, therefore `??` operator is enough to fallback to `allReportActions` if the param is not passed.

Let me know if you have a problem with the `altReportActions` naming and what suggestions you'd have if so.
Otherwise let me know what else is required for you to approve this.

Reply:
Oh, I am blind, did not notice the `t` there 🤦 

Yeah, that's what I was suggesting

Reply:
@iwiznia Done, removed `isEmptyObject` check and changed it to:

`const reportActions = (altReportActions ?? allReportActions)?.['${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}'] ?? {};`

Works as expected!